src/WebJobs.Extensions.DurableTask/Listener/TaskOrchestrationShim.cs

```diff
@@ -18,10 +18,12 @@ internal class TaskOrchestrationShim : TaskCommonShim
     {
         private readonly DurableOrchestrationContext context;
         private readonly OutOfProcOrchestrationShim outOfProcShim;
+        private readonly DurableTaskExtension config;

```

Comment:
Instead of passing in the whole extension config, can you just pass in the dataconverter?